src/pages/Roadmap.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { tsx } from '@dojo/framework/widget-core/tsx';
+import harness from '@dojo/framework/testing/harness';
+
+import Card from '../widgets/card/Card';
+import CardHeader from '../widgets/card/CardHeader';
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import LocalPage from '../widgets/page/LocalPage';
+
+import Roadmap from './Roadmap';
+import * as css from './Roadmap.m.css';
+
+describe('Roadmap Page', () => {
+	it('renders', () => {
+		const h = harness(() => <Roadmap />);
+
+		h.expectPartial('@timeline > *:last-child', () => (
```

Comment:
I've never seen this syntax before. What does `@timline > <some_selector>` do?

Reply:
Its searching for the dojo `key` `timeline` (aka `@timeline`), then looking at its children. In this case, the last child. It is a mix of CSS and Dojo selection supported by the harness API.